Library/Formula/sfml.rb

```diff
@@ -27,9 +17,45 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
```

Comment:
Interesting discussion in that thread. Looks good, whatever way you decide to implement it. If we get a firm implementation, we can either backport the change as a ` .diff ` until the next release. Generally speaking, if something is committed upstream, we're happy to use it in-formula until the next release makes it redundant.